py/torch_tensorrt/dynamo/aten_tracer.py

```diff
@@ -85,7 +87,6 @@ def setting_python_recursive_limit(limit: int = 10000) -> Generator[None, None,
         sys.setrecursionlimit(default)
 
 
-@req_torch_version("2.dev")
```

Comment:
These function wrappers are intended to warn users compiling with the legacy library that these functions will not work with Torch 1.13. Are they removed here to be replaced by alternative log messaging?

Reply:
So mypy complains about these decorators. I've changed the decorators according to mypy conventions. This should be resolved now.